### PR TITLE
Add speakers to dialog lines

### PR DESCRIPTION
**Pull request by KelseyHigham**
I added some of these by guessing; some of them by looking at the game; and some of them by changing data and seeing what changed. I tried to indicate when I didn't know something, so I don't think there are any errors. 

- I labeled "Narrator" lines to distinguish them from unlabeled lines. 
- I labeled "idk" lines to distinguish them from unlabeled lines. 
- I labeled player responses to questions as "Link". 
- I used official names from the Switch version where I knew them. 
- I only labeled English lines. I'm not sure whether the translations should use English character names, or localized ones.

This is kind of messy, but useful to translators, I think.

### CODE REVIEW

File: src/text/dialog_1.asm [hunk 1]
```diff
@@ -43,21 +43,21 @@ Dialog003::
     db "'Ballad of the  "
     db "Wind Fish.'@"
 
-Dialog004::
+Dialog004:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:46]: Yes, this is Marin
```suggestion
Dialog004:: ; Marin
```

File: src/text/dialog_1.asm [hunk 2]
```diff
@@ -170,15 +170,15 @@ Dialog012::
     db "the wall with a "
     db "Bomb!@"
 
-Dialog013::
+Dialog013:: ; Narrator, probably
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:173]:
```suggestion
Dialog013:: ; Narrator
```

File: src/text/dialog_1.asm [hunk 3]
```diff
@@ -170,15 +170,15 @@ Dialog012::
...
     db "You've learned  "
     db "the 'Ballad of  "
     db "the Wind Fish!' "
     db "This song will  "
     db "always remain in"
     db "your heart!@"
 
-Dialog014::
+Dialog014:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:181]:
```suggestion
Dialog014:: ; Marin
```

File: src/text/dialog_1.asm [hunk 4]
```diff
@@ -187,13 +187,13 @@ Dialog014::
     db "fresh in your   "
     db "mind!@"
 
-Dialog015::
+Dialog015:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:190]:
```suggestion
Dialog015:: ; Marin
```

File: src/text/dialog_1.asm [hunk 5]
```diff
@@ -187,13 +187,13 @@ Dialog014::
...
     db "Please!  I want "
     db "you to learn it!"
     db "This song is my "
     db "favorite!@"
 
-Dialog016::
+Dialog016:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:196]:
```suggestion
Dialog016:: ; Marin
```

File: src/text/dialog_1.asm [hunk 6]
```diff
@@ -239,78 +239,78 @@ ENDC
     db "Drop by again,  "
     db "big guy!@"
 
-Dialog01B::
+Dialog01B:: ; Crazy Tracy, probably
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:242]:
```suggestion
Dialog01B:: ; Crazy Tracy
```

File: src/text/dialog_1.asm [hunk 7]
```diff
@@ -239,78 +239,78 @@ ENDC
...
     db "cash!@"
 
-Dialog01C::
+Dialog01C:: ; Crazy Tracy
     db "... ... ... ... "
     db "But I won't sell"
     db "it to you!@"
 
-Dialog01D::
+Dialog01D:: ; Crazy Tracy, probably
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:253]:
```suggestion
Dialog01D:: ; Crazy Tracy
```

File: src/text/dialog_1.asm [hunk 8]
```diff
@@ -239,78 +239,78 @@ ENDC
...
     db "discount!@"
 
-Dialog01F::
+Dialog01F:: ; Narrator
     db "You got Marin!  "
     db "Is this your big"
     db "chance?@"
 
-Dialog020::
+Dialog020:: ; Fox
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:272]: It looks like a fox, but is actually a dog! https://zelda.fandom.com/wiki/Mutt
```suggestion
Dialog020:: ; Mutt
```

File: src/text/dialog_1.asm [hunk 9]
```diff
@@ -320,112 +320,112 @@ Dialog027::
     db "keep a look out "
     db "for me, hear?@"
 
-Dialog028::
+Dialog028:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:323]:
```suggestion
Dialog028:: ; Marin
```

File: src/text/dialog_1.asm [hunk 10]
```diff
@@ -621,232 +621,232 @@ Dialog055::
...
+Dialog083:: ; Signpost
     db "Telephone Booth "
     db "<down> Signpost Maze @"
 
-Dialog084::
+Dialog084:: ; Map
     db "Seashell Mansion@"
 
-Dialog085::
+Dialog085:: ; Map, probably
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:782]:
```suggestion
Dialog085:: ; Map
```

File: src/text/dialog_1.asm [hunk 11]
```diff
@@ -621,232 +621,232 @@ Dialog055::
...
+Dialog08D:: ; Narrator
     db "Wow!  This looks"
     db "pretty heavy!   "
     db "You won't be    "
     db "able to lift it "
     db "with just your  "
     db "bare hands...@"
 
-Dialog08E::
+Dialog08E:: ; Narrator, probably
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:825]:
```suggestion
Dialog08E:: ; Narrator
```

File: src/text/dialog_1.asm [hunk 12]
```diff
@@ -621,232 +621,232 @@ Dialog055::
...
     db "Well, it's an   "
     db "Ocarina, but you"
     db "don't know how  "
     db "to play it...@"
 
-Dialog08F::
+Dialog08F:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:831]:
```suggestion
Dialog08F:: ; Marin
```

File: src/text/dialog_1.asm [hunk 13]
```diff
@@ -621,232 +621,232 @@ Dialog055::
...
-Dialog091::
+Dialog091:: ; Narrator
     db "You got your    "
     db "shield back!    "
     db "Press the button"
     db "and repel       "
     db "enemies with it!@"
 
-Dialog092::
+Dialog092:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:849]: https://zelda.fandom.com/wiki/%3F%3F%3F_(Link%27s_Awakening)
```suggestion
Dialog092:: ; Desert skull
```

File: src/text/dialog_1.asm [hunk 14]
```diff
@@ -860,109 +860,109 @@ Dialog092::
...
-Dialog099::
+Dialog099:: ; Narrator
     db "You've got some "
     db "Magic Powder!   "
     db "Try sprinkling  "
     db "it on a variety "
     db "of things!@"
 
-Dialog09A::
+Dialog09A:: ; Narrator, probably
```
Comment (tobiasvl) [inline src/text/dialog_1.asm:910]:
```suggestion
Dialog09A:: ; Narrator
```

File: src/text/dialog_2.asm [hunk 15]
```diff
@@ -26,84 +26,84 @@ Dialog0A7::
...
 
-Dialog0B4::
+Dialog0B4:: ; Genie (D2 Nightmare)
     db "HO HO HO!       "
     db "I'm your bad guy"
     db "this time!!     "
     db "HO HO HO!@"
 
-Dialog0B5::
+Dialog0B5:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_2.asm:106]:
```suggestion
Dialog0B5:: ; Slime Eel (D5 Nightmare)
```

File: src/text/dialog_2.asm [hunk 16]
```diff
@@ -113,15 +113,15 @@ Dialog0B5::
...
+Dialog0B6:: ; Facade (D6 Nightmare)
     db "Hey dummy! Need "
     db "a hint?  My weak"
     db "point is... !!  "
     db "Whoops!  There I"
     db "go, talking too "
     db "much again...@"
 
-Dialog0B7::
+Dialog0B7:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_2.asm:124]:
```suggestion
Dialog0B7:: ; Facade (D6 Nightmare)
```

File: src/text/dialog_2.asm [hunk 17]
```diff
@@ -482,34 +482,34 @@ Dialog0D0::
     db "LET US AWAKEN..."
     db "   TOGETHER!!@"
 
-Dialog0D1::
-Dialog0D2::
+Dialog0D1:: ; Wind Fish, probably
```
Comment (tobiasvl) [inline src/text/dialog_2.asm:485]:
```suggestion
Dialog0D1:: ; Wind Fish
```

File: src/text/dialog_2.asm [hunk 18]
```diff
@@ -482,34 +482,34 @@ Dialog0D0::
...
+Dialog0D2:: ; Wind Fish, probably
```
Comment (tobiasvl) [inline src/text/dialog_2.asm:486]:
```suggestion
Dialog0D2:: ; Wind Fish
```

File: src/text/dialog_2.asm [hunk 19]
```diff
@@ -482,34 +482,34 @@ Dialog0D0::
...
-Dialog0D4::
-Dialog0D5::
+Dialog0D3:: ; Map, probably
+Dialog0D4:: ; Map, probably
+Dialog0D5:: ; Map, probably
     db "Mermaid Statue  @"
 
-Dialog0D6::
-Dialog0D7::
+Dialog0D6:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_2.asm:497]:
```suggestion
Dialog0D6:: ; Owl
```

File: src/text/dialog_2.asm [hunk 20]
```diff
@@ -482,34 +482,34 @@ Dialog0D0::
...
+Dialog0D7:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_2.asm:498]:
```suggestion
Dialog0D7:: ; Owl
```

File: src/text/dialog_2.asm [hunk 21]
```diff
@@ -533,12 +533,12 @@ Dialog0D9::
     db "wait for you    "
     db "there!  Hoot!@"
 
-Dialog0DA::
+Dialog0DA:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_2.asm:536]:
```suggestion
Dialog0DA:: ; Lanmola
```

File: src/text/dialog_3.asm [hunk 22]
```diff
@@ -293,7 +293,7 @@ Dialog12D::
     db "we can talk!    "
     db "Good Bye!@"
 
-Dialog12E::
+Dialog12E:: ; Photographer, probably
```
Comment (tobiasvl) [inline src/text/dialog_3.asm:296]:
```suggestion
Dialog12E:: ; Photographer
```

File: src/text/dialog_3.asm [hunk 23]
```diff
@@ -1073,12 +1073,12 @@ Dialog189::
...
     db "Cha-cha-cha!@"
 
-Dialog18A::
+Dialog18A:: ; Manbo
     db "Aha... You don't"
     db "have an Ocarina,"
     db "so...Glub glub!@"
 
-Dialog18B::
+Dialog18B:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_3.asm:1081]:
```suggestion
Dialog18B:: ; Henhouse Keeper
```

File: src/text/dialog_3.asm [hunk 24]
```diff
@@ -1089,7 +1089,7 @@ Dialog18B::
     db "But now, see?   "
     db "Cluck cluck!@"
 
-Dialog18C::
+Dialog18C:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_3.asm:1092]:
```suggestion
Dialog18C:: ; Henhouse Keeper
```

File: src/text/dialog_3.asm [hunk 25]
```diff
@@ -1100,17 +1100,17 @@ Dialog18C::
     db "your head? Cluck"
     db "Cluck!@"
 
-Dialog18D::
+Dialog18D:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_3.asm:1103]:
```suggestion
Dialog18D:: ; Henhouse Keeper
```

File: src/text/dialog_4.asm [hunk 26]
```diff
@@ -28,12 +28,12 @@ Dialog192::
     db "what is your    "
     db "favorite song?@"
 
-Dialog193::
+Dialog193:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:31]:
```suggestion
Dialog193:: ; Marin
```

File: src/text/dialog_4.asm [hunk 27]
```diff
@@ -28,12 +28,12 @@ Dialog192::
...
     db "Please, don't   "
     db "ever forget this"
     db "song...or me...@"
 
-Dialog194::
+Dialog194:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:36]:
```suggestion
Dialog194:: ; Marin
```

File: src/text/dialog_4.asm [hunk 28]
```diff
@@ -57,13 +57,13 @@ Dialog195::
     db "never forgive   "
     db "you!@"
 
-Dialog196::
+Dialog196:: ; Fox
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:60]:
```suggestion
Dialog196:: ; Mutt, narrator
```

File: src/text/dialog_4.asm [hunk 29]
```diff
@@ -57,13 +57,13 @@ Dialog195::
...
     db "... ... ... ... "
     db "It seems to be  "
     db "totally absorbed"
     db "in Marin's song!@"
 
-Dialog197::
+Dialog197:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:66]:
```suggestion
Dialog197:: ; Marin
```

File: src/text/dialog_4.asm [hunk 30]
```diff
@@ -74,26 +74,26 @@ Dialog197::
     db "he make my wish "
     db "come true?@"
 
-Dialog198::
+Dialog198:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:77]:
```suggestion
Dialog198:: ; Marin
```

File: src/text/dialog_4.asm [hunk 31]
```diff
@@ -74,26 +74,26 @@ Dialog197::
...
     db "Eh?  You want me"
     db "to go in there? "
     db "No, I think I'll"
     db "wait out here..."
     db "Take care of    "
     db "yourself, #####!@"
 
-Dialog199::
+Dialog199:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:85]:
```suggestion
Dialog199:: ; Marin
```

File: src/text/dialog_4.asm [hunk 32]
```diff
@@ -74,26 +74,26 @@ Dialog197::
...
     db "Ahhh!  Ahhh, you"
     db "are a bad boy,  "
     db "#####!@"
 
-Dialog19A::
+Dialog19A:: ; idk, maybe Crazy Tracy?
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:90]:
```suggestion
Dialog19A:: ; Crazy Tracy
```

File: src/text/dialog_4.asm [hunk 33]
```diff
@@ -107,194 +107,194 @@ Dialog19B::
...
-Dialog1C3::
+Dialog1C3:: ; Signpost
     db "Beware of Sea   "
     db "Urchins!  Don't "
     db "touch them with "
     db "your bare hands!@"
 
-Dialog1C4::
-Dialog1C5::
+Dialog1C4:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:296]:
```suggestion
Dialog1C4:: ; Tarin
```

File: src/text/dialog_4.asm [hunk 34]
```diff
@@ -107,194 +107,194 @@ Dialog19B::
...
+Dialog1C5:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:297]:
```suggestion
Dialog1C5:: ; Tarin
```

File: src/text/dialog_4.asm [hunk 35]
```diff
@@ -421,21 +421,21 @@ Dialog1D2::
...
+Dialog1D4:: ; Chef Bear
     db "HI HO!  Little  "
     db "Marin!  Welcome!"
     db "... ... ... ... "
     db "Oh, shucks!  You"
     db "are here too... "
     db "Sorry...@"
 
-Dialog1D5::
+Dialog1D5:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:438]:
```suggestion
Dialog1D5:: ; Marin
```

File: src/text/dialog_4.asm [hunk 36]
```diff
@@ -444,16 +444,16 @@ Dialog1D5::
     db "for a while?    "
     db "    Yes! No...<ask>"
 
-Dialog1D6::
+Dialog1D6:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:447]:
```suggestion
Dialog1D6:: ; Marin
```

File: src/text/dialog_4.asm [hunk 37]
```diff
@@ -488,90 +488,90 @@ Dialog1D9::
...
 
-Dialog1DB::
+Dialog1DB:: ; Marin
     db "I want to know  "
     db "everything about"
     db "you...Err...Uhh,"
     db "Ha ha ha ha!@"
 
-Dialog1DC::
+Dialog1DC:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:504]:
```suggestion
Dialog1DC:: ; Marin
```

File: src/text/dialog_4.asm [hunk 38]
```diff
@@ -488,90 +488,90 @@ Dialog1D9::
...
     db "Hunh? The walrus"
     db "wants me to go  "
     db "to him?  It     "
     db "doesn't matter, "
     db "I will go with  "
     db "you to him...@"
 
-Dialog1DD::
+Dialog1DD:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:512]:
```suggestion
Dialog1DD:: ; Tarin
```

File: src/text/dialog_4.asm [hunk 39]
```diff
@@ -488,90 +488,90 @@ Dialog1D9::
...
     db "Unnnngh! Owwwww!"
     db "... ... ... ... "
     db "I've sure lost  "
     db "my taste for    "
     db "honey!@"
 
-Dialog1DE::
+Dialog1DE:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:519]:
```suggestion
Dialog1DE:: ; Marin
```

File: src/text/dialog_4.asm [hunk 40]
```diff
@@ -488,90 +488,90 @@ Dialog1D9::
...
 
-Dialog1DF::
+Dialog1DF:: ; idk
     db "Ha! That's all  "
     db "you've got?!    "
     db "Get ready for   "
     db "THIS!@"
 
-Dialog1E0::
+Dialog1E0:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:532]:
```suggestion
Dialog1E0:: ; Walrus
```

File: src/text/dialog_4.asm [hunk 41]
```diff
@@ -488,90 +488,90 @@ Dialog1D9::
...
     db "ZZZ ZZZ ZZZ ZZZ "
     db " ... <marin> ... <marin> ...@"
 
-Dialog1E1::
+Dialog1E1:: ; Marin, probably, and Link
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:536]:
```suggestion
Dialog1E1:: ; Marin, Link
```

File: src/text/dialog_4.asm [hunk 42]
```diff
@@ -488,90 +488,90 @@ Dialog1D9::
...
     db "Yes, it's that  "
     db "lazy walrus!    "
     db "Shall we give   "
     db "him a little    "
     db "surprise?       "
     db "    Yes  No...<ask>"
 
-Dialog1E2::
+Dialog1E2:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:544]:
```suggestion
Dialog1E2:: ; Marin
```

File: src/text/dialog_4.asm [hunk 43]
```diff
@@ -488,90 +488,90 @@ Dialog1D9::
...
 
-Dialog1E3::
+Dialog1E3:: ; idk
     db "Hunh?  Oh, he's "
     db "calling me...   "
     db "It's the same as"
     db "always... Ha ha!@"
 
-Dialog1E4::
+Dialog1E4:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:556]:
```suggestion
Dialog1E4:: ; Marin
```

File: src/text/dialog_4.asm [hunk 44]
```diff
@@ -488,90 +488,90 @@ Dialog1D9::
...
 
-Dialog1E2::
+Dialog1E2:: ; Marin, probably
     db "Aha ha ha!  Wow!"
     db "He certainly    "
     db "woke with a     "
     db "start!@"
 
-Dialog1E3::
+Dialog1E3:: ; idk
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:550]:
```suggestion
Dialog1E3:: ; Marin
```

File: src/text/dialog_4.asm [hunk 45]
```diff
@@ -596,38 +596,38 @@ Dialog1E8::
     db "me have it...   "
     db "    Okay No<ask>"
 
-Dialog1E9::
+Dialog1E9:: ; Fisherman, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:599]:
```suggestion
Dialog1E9:: ; Fisherman
```

File: src/text/dialog_4.asm [hunk 46]
```diff
@@ -596,38 +596,38 @@ Dialog1E8::
...
 
-Dialog1EA::
+Dialog1EA:: ; Fisherman, probably
     db "You should be   "
     db "more kind to me!"
     db "I thought we    "
     db "were buddies!@"
 
-Dialog1EB::
+Dialog1EB:: ; Fisherman, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:610]:
```suggestion
Dialog1EB:: ; Fisherman
```

File: src/text/dialog_4.asm [hunk 47]
```diff
@@ -596,38 +596,38 @@ Dialog1E8::
...
     db "BIIIIG one!@"
 
-Dialog1EC::
+Dialog1EC:: ; Narrator
     db "The <fishhook> became a  "
     db "necklace <bra>!     "
     db "L-l-lucky!@"
 
-Dialog1ED::
+Dialog1ED:: ; Fisherman, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:619]:
```suggestion
Dialog1ED:: ; Fisherman
```

File: src/text/dialog_4.asm [hunk 48]
```diff
@@ -596,38 +596,38 @@ Dialog1E8::
...
     db "I can't wait to "
     db "see what I'll   "
     db "catch next!@"
 
-Dialog1EE::
+Dialog1EE:: ; Mamasha, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:624]:
```suggestion
Dialog1EE:: ; Mamasha
```

File: src/text/dialog_4.asm [hunk 49]
```diff
@@ -596,38 +596,38 @@ Dialog1E8::
...
     db "My husband is   "
     db "lost in the     "
     db "woods! Please   "
     db "go find him!@"
 
-Dialog1EF::
+Dialog1EF:: ; Secret Zora or Secret Goriya, not sure
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:630]:
```suggestion
Dialog1EF:: ; Secret Zora
```

File: src/text/dialog_4.asm [hunk 50]
```diff
@@ -663,28 +663,28 @@ Dialog1F2::
     db "scale as I said!"
     db "    Give Keep<ask>"
 
-Dialog1F3::
+Dialog1F3:: ; Mermaid
     db "Promise!  You'll"
     db "only take one!@"
 
-Dialog1F4::
+Dialog1F4:: ; Mermaid, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:670]:
```suggestion
Dialog1F4:: ; Mermaid
```

File: src/text/dialog_4.asm [hunk 51]
```diff
@@ -994,32 +994,32 @@ Dialog21A::
     db "this book reeks "
     db "of secrets...@"
 
-Dialog21B::
+Dialog21B:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:997]:
```suggestion
Dialog21B:: ; Marin
```

File: src/text/dialog_4.asm [hunk 52]
```diff
@@ -994,32 +994,32 @@ Dialog21A::
...
     db "...You're late! "
     db "I thought you'd "
     db "never come back!@"
 
-Dialog21C::
+Dialog21C:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:1002]:
```suggestion
Dialog21C:: ; Marin
```

File: src/text/dialog_4.asm [hunk 53]
```diff
@@ -994,32 +994,32 @@ Dialog21A::
...
     db "...EEEK!  You're"
     db "hurt!  Arrrgh!  "
     db "Don't be so     "
     db "reckless!@"
 
-Dialog21D::
+Dialog21D:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_4.asm:1008]:
```suggestion
Dialog21D:: ; Marin
```

File: src/text/dialog_dx.asm [hunk 54]
```diff
@@ -42,29 +42,29 @@ Dialog253::
...
     db "cowards.@"
 
-Dialog254::
+Dialog254:: ; idk
     db "I'm not afraid. "
     db "I just decided  "
     db "to wait at home.@"
 
-Dialog255::
+Dialog255:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_dx.asm:50]:
```suggestion
Dialog255:: ; Marin
```

File: src/text/dialog_dx.asm [hunk 55]
```diff
@@ -42,29 +42,29 @@ Dialog253::
...
     db "This is my first"
     db "walk with you,  "
     db "#####.@"
 
-Dialog256::
+Dialog256:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_dx.asm:55]:
```suggestion
Dialog256:: ; Marin
```

File: src/text/dialog_dx.asm [hunk 56]
```diff
@@ -42,29 +42,29 @@ Dialog253::
...
     db ". . . . .@"
 
-Dialog257::
+Dialog257:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_dx.asm:58]:
```suggestion
Dialog257:: ; Marin
```

File: src/text/dialog_dx.asm [hunk 57]
```diff
@@ -42,29 +42,29 @@ Dialog253::
...
     db "This cliff will "
     db "be our secret   "
     db "place. @"
 
-Dialog258::
+Dialog258:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_dx.asm:63]:
```suggestion
Dialog258:: ; Marin
```

File: src/text/dialog_dx.asm [hunk 58]
```diff
@@ -205,254 +205,254 @@ Dialog26A::
...
 
-Dialog278::
+Dialog278:: ; Marin
     db "#####, do you   "
     db "always look in  "
     db "other people's  "
     db "drawers?@"
 
-Dialog279::
+Dialog279:: ; Marin, probably
```
Comment (tobiasvl) [inline src/text/dialog_dx.asm:282]:
```suggestion
Dialog279:: ; Marin
```

**Review by tobiasvl — CHANGES_REQUESTED** [review]

File: src/text/dialog_1.asm (hunk 8 above)
Comment (KelseyHigham) [reply src/text/dialog_1.asm:272]: interesting! it's a fox in the Japanese version:
https://lizbushouse.com/links-awakening-script-comparison-mabe-village/

**Comment by KelseyHigham** [comment]
I think I addressed all changes and marked all conversations as resolved, but GitHub is still telling me this:
> Merging is blocked
> Merging can be performed automatically once the requested changes are addressed.

I think this can be merged now.

**Comment by kemenaran** [comment]
@tobiasvl I think you need to approve the PR to allow it to be merged

**Review by tobiasvl — APPROVED** [review]

**Comment by tobiasvl** [comment]
Ah, right, sorry. I guess I didn't notice it because my review wasn't manually re-requested. LGTM!

**Pull request merged**